src/plugins/synced-lyrics/renderer/lyrics/fetch.ts

```diff
@@ -178,10 +178,12 @@ export const getLyricsList = async (
   // Add a blank line at the beginning
   raw.unshift('[0:0.0] ');
 
-  const syncedLyricList = raw.reduce<LineLyrics[]>((acc, line, index) => {
-    const syncedLine = extractTimeAndText(line, index);
+  let i = 0;
```

Comment:
Why did you change this line?

Reply:
raw.reduce doesn't account for `acc`'s length. If we don't push a line, `index` would still be increased by 1.